mesa/visualization/mpl_space_drawing.py

```diff
@@ -175,13 +191,34 @@ def draw_property_layers(
         so you can do `{"some_layer":{"colormap":'viridis', 'alpha':.25, "colorbar":False}}`
 
     """
+
+    def get_hexmesh(width: int, height: int) -> list[list[tuple[float, float]]]:
+        """Create hexagon vertices for the mesh."""
+        hexagons = []
+        size = 1
+        x_spacing = np.sqrt(3) * size
+        y_spacing = 1.5 * size
+
+        for row in range(height):
+            for col in range(width):
+                # Calculate center position with offset for even rows
+                x = col * x_spacing + (row % 2 == 0) * (x_spacing / 2)
+                y = row * y_spacing
+                vertices = _get_hex_vertices(x, y, size)
```

Comment:
This is repetetive with lines 417 - 420. Might it be possible to add an additional helper method that returns all vertices for a given hexgrid? If so, we can even cash this using `functools.cache` for a small speedup when plotting.